sdk/src/test/java/io/opentelemetry/sdk/metrics/MeterSdkTest.java

```diff
@@ -75,6 +75,7 @@ public void testLabelSets() {
     assertThat(testSdk.createLabelSet()).isSameInstanceAs(testSdk.createLabelSet());
     assertThat(testSdk.createLabelSet())
         .isSameInstanceAs(testSdk.createLabelSet(Collections.<String, String>emptyMap()));
+    assertThat(testSdk.createLabelSet()).isNotNull();
```

Comment:
huh. I would have guessed that `isSameInstanceAs` would have included it being not null. This is totally fine, though.

Reply:
they are the same instance "null" :) As you can see in my fix that was null :)